data/cookbook/serialise-to-json/00-serialize-yojson.ml

```diff
@@ -0,0 +1,27 @@
+---
+packages: 
+- name: yojson
+  tested_version: 2.1.2
+  used_libraries:
+  - ppx_yojson_conv
+---
+(* We need to open Ppx_yojson_conv_lib.Yojson_conv.Primitives in order to serialize values of primitive types. *)
+open Ppx_yojson_conv_lib.Yojson_conv.Primitives
+
+type language = { 
+  name: string; 
+  url: string 
+} [@@deriving yojson]
+
+(* First we convert the json string to Yojson.Safe.t and then we use the generated function to create the record *)
+(* If the parsing fails the result will be None. Otherwise it will be the deserialized record*)
```

Comment:
```suggestion
(* First we convert the json string to Yojson.Safe.t and then we use the generated function to create the record.
   If the parsing fails the result will be None. Otherwise it will be the deserialized record*)
```

Reply:
no worries, thanks for the guidance